typings/react-native-background-task/index.d.ts

```diff
@@ -0,0 +1,21 @@
+declare module 'react-native-background-task' {
```

Comment:
In this `typings` folder, we can add TS definitions for modules that don't have TS definitions available.